lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -700,17 +701,16 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
     err_rows = [["id", "username", "error_msg"]]
     current_step = {'step': 'Calculating Grades'}
 
-    total_enrolled_students = enrolled_students.count()
     student_counter = 0
     TASK_LOG.info(
         u'%s, Task type: %s, Current step: %s, Starting grade calculation for total students: %s',
         task_info_string,
         action_name,
         current_step,
-
-        total_enrolled_students
+        total_enrolled_students,
     )
-    for student, gradeset, err_msg in iterate_grades_for(course_id, enrolled_students):
+    # NAA TODO - Get the list of ALL subsections to put in the header
```

Comment:
As discussed, I'm going to treat this as a feature branch and address this in a subsequent PR.